packages/interface-compliance-tests/src/connection/index.ts

```diff
@@ -127,7 +127,7 @@ export default (test: TestSetup<Connection>): void => {
       it.skip('should track inbound streams', async () => {
         // Add an remotely opened stream
         const stream = stubInterface<Stream>()
-        connection.addStream(stream)
+        stream.direction = 'inbound'
```

Comment:
all .addStream was doing here is `stream.direction = 'inbound'`. can this test just be removed since it was already skipped?

Reply:
Yes this can be removed